From cba68daa64474c1bb5df91852674e7e38e44bba0 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Thu, 10 Nov 2022 09:41:50 +0100
Subject: [PATCH 086/529] ARM: configs: enable SCMI voltage domain in STM32MP1
 addons fragment

Enable SCMI voltage domain in local Linux config fragment.

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/225906
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
Tested-by: Pascal PAILLET <p.paillet@st.com>
Change-Id: Ia570a265fe3451b709ecc450f65c4633cf2ed528
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/275802
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -1,6 +1,9 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-multiv7_addons.config -o fragment-02-multiv7_addons.config
 CONFIG_AEABI=y
+CONFIG_ARM_SCMI_PROTOCOL=y
+CONFIG_ARM_SMC_MBOX=y
 CONFIG_CHR_DEV_SG=y
+CONFIG_COMMON_CLK_SCMI=y
 CONFIG_CRC_ITU_T=m
 CONFIG_DEBUG_INFO=y
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
@@ -31,6 +34,7 @@ CONFIG_POWER_RESET_SYSCON=y
 CONFIG_PREEMPT=y
 CONFIG_PREEMPT_COUNT=y
 CONFIG_PROFILING=y
+CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_SCHED_MC=y
 CONFIG_SENSORS_IIO_HWMON=y
 CONFIG_SERIAL_NONSTANDARD=y
